core/src/main/java/com/godaddy/commerce/core/http/HttpClientCache.java

```diff
@@ -110,6 +111,9 @@ private static CloseableHttpClient createClient(CacheKey key) {
       builder.setDefaultCredentialsProvider(provider);
     }
 
+    if(key.httpClientBuilderCustomizer != null)
+      key.httpClientBuilderCustomizer.accept(builder);
```

Comment:
> [<img alt="mtbennett" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.secureserver.net/mtbennett) **Authored by [mtbennett](https://github.secureserver.net/mtbennett)**
_<time datetime="2022-10-28T15:54:14Z" title="Friday, October 28th 2022, 5:54:14 pm +02:00">Oct 28, 2022</time>_

---


```suggestion
    if (key.httpClientBuilderCustomizer != null) {
      key.httpClientBuilderCustomizer.accept(builder);
    }
```